### DISCUSSION TITLE
Ice team?

### DISCUSSION THREAD

**Discussion by indicum** [ice-blockchain/mobile-app#49] [General]
Hi,
In https://ice.io/faq, it's stated:

_Our team’s work can be seen on [GitHub](https://github.com/ice-blockchain) in a very transparent approach._

How can e see the Ice Team?

**Comment by allbyte** [comment]
You can see the team's work by initial nickname @ice-zeus @ice-ares @ice-aiolos @ice-hector @ice-orion @ice-hades @ice-atlas @ice-myles @ice-cronus

**Reply by Genius8it** [reply]
Okay

**Comment by helka41** [comment]
Hello, everybody! I have read on your site that it will be possible to exchange the accumulated ice coins into crypto currencies. Will I be able to exchange them into Bitcoins? Will Ice network be able to covert the ice coins into usual bank currencies 💵 in the next years? Will it become possible to transfer the ice coins onto my bank card or PayPal?

**Reply by Genius8it** [reply]
Yah ice network will be listed on October 7 2024 and it will be exchange to other currencies and cryptos like bitcoin and ethereum on  market places like Binance and etc

**Reply by Genius8it** [reply]
Ice coin mining is free using your mobile phone. DOWNLOAD the app at playstore or AppStore.Use my invitation code( Genius8it) to receive 10 ice coins and continue mining

**Reply by helka41** [reply]
Thank you very much for your reply. I have already joined the ICE network

**Comment by helka41** [comment]
Hi, there! Something is wrong with my prestaking setting. When I check ✅ the green square box and agree with the prestaking conditions, the prestaking button, which is below, does not want to be activated. It's not active. I have adjusted my setting for 5 years and distribution for 100%. But the button is not active. Is that a problem? Or, perhaps, that means the settings are at their maximum, does not it? Maybe, I prestaked too much, did not I?

**Reply by Genius8it** [reply]
Try to reduce the distribution

**Reply by helka41** [reply]
How to do it?

**Reply by Genius8it** [reply]
If u cannot adjust now just stick to previous selected settings the app will be updated later and everything will be fine
